onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java

```diff
@@ -81,10 +82,14 @@ public static void show(AppCompatActivity activity, Integer containerViewId,
         fragment.setArguments(FragmentUtils.getIntentArgs(intent));
         fragment.setCallback(callback);
 
-        FragmentTransaction ft = fm.beginTransaction();
-        ft.replace(containerViewId, fragment, TAG);
-        ft.addToBackStack(null);
-        ft.commit();
+        try {
+            FragmentTransaction ft = fm.beginTransaction();
+            ft.replace(containerViewId, fragment, TAG);
+            ft.addToBackStack(null);
+            ft.commit();
+        } catch (IllegalStateException e) {
+            Log.v("ArrivalListFragment", "Cannot show SimpleArrivalListFragment");
```

Comment:
Ah - yes, I think that's Lint, I've seen it before.  In that case, just shorten the TAG to be `SimpArrivalListFragment` - I've done that with a few other Fragments/Activities.